app/services/foreman_rh_cloud/ui_request_forwarder.rb

```diff
@@ -0,0 +1,106 @@
+require 'rest-client'
+
+module ForemanRhCloud
+  class UIRequestForwarder
```

Comment:
The requests that are being forwarded are for Insights APIs. I'd suggest to consider some other naming, like `InsightsRequestForwarder`, `InsightsAPIRequestForwarder`, `InsightsAPIForwarder`...